xcodeproj/internal/xcschemes/xcschemes.bzl

```diff
@@ -361,6 +361,7 @@ def _test(
         diagnostics = None,
         env = "inherit",
         env_include_defaults = True,
+        test_options = None,
```

Comment:
Sorry for message on the PR instead of creating issue, Im getting this failure on `2.9.0`, and suspecting this is the cause 

> Error in add_all: expected value of type 'sequence or depset' for values, got 'NoneType'

```
	File "/bazel-output-base/rules_xcodeproj.noindex/build_output_base/external/rules_xcodeproj~2.9.0/xcodeproj/internal/xcodeproj_incremental_rule.bzl", line 708, column 53, in _xcodeproj_incremental_impl
		(xcschemes, xcschememanagement) = _write_schemes(
	File "/bazel-output-base/rules_xcodeproj.noindex/build_output_base/external/rules_xcodeproj~2.9.0/xcodeproj/internal/xcodeproj_incremental_rule.bzl", line 540, column 67, in _write_schemes
		autogeneration_config_file = _write_autogeneration_config_file(
	File "/bazel-output-base/rules_xcodeproj.noindex/build_output_base/external/rules_xcodeproj~2.9.0/xcodeproj/internal/xcodeproj_incremental_rule.bzl", line 222, column 17, in _write_autogeneration_config_file
		args.add_all(config.get("test_options"))
```

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3108